File: C#/2019-2-C샵-5주차-배열이용하기-김유빈/2019-2-C샵-5주차-배열이용하기-김유빈/Form3.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _2019_2_C샵_5주차_배열이용하기_김유빈
{
    public partial class Form3 : Form
    {
        public Form3()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            for (int i = 1; i < 10; i = i + 3)
            {
                for (int j = 1; j < 10; j++)
                {
                    for (int k = 0; k < 3; k++)
                    {
                        textBox1.Text = textBox1.Text + (i + k) + " X " + j + " = " + (i + k) * j + "     ";
                    }
                    textBox1.Text = textBox1.Text + "\r\n";
                }
                textBox1.Text = textBox1.Text + "\r\n";
            }
        }
    }
}

File: 비주얼 베이직/중간고사/1/1/Form3.vb
Public Class Form3
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim i, j As Integer
        TextBox1.Text = ""
        For i = 1 To 9 Step 1
            For j = 1 To 9 Step 1
                TextBox1.Text = TextBox1.Text & i & " X " & j & " = " & (i * j) & vbCrLf
            Next
            TextBox1.Text = TextBox1.Text & vbCrLf

        Next

    End Sub
End Class